packages/ws/src/strategies/sharding/WorkerShardingStrategy.ts

```diff
@@ -1,5 +1,5 @@
 import { once } from 'node:events';
-import { join } from 'node:path';
+import path, { join } from 'node:path';
```

Comment:
Why are you importing path instead of its individual methods? Considering you do that already for `join`